### PR TITLE
Enable EOS storage commands on ocis-reva

### PR DESCRIPTION
**Pull request by diocas**

### CODE REVIEW

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ pkg/command/storageeos.go  -2
+ pkg/command/storageeosdata.go  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=5426902&bid=17796625)

File: pkg/command/storageeos.go [hunk 1]
```diff
@@ -108,24 +108,6 @@ func StorageEOS(cfg *config.Config) *cli.Command {
 										"enable_home":            cfg.Reva.Storages.EOS.EnableHome,
 										"user_layout":            cfg.Reva.Storages.EOS.Layout,
 									},
-									"local": map[string]interface{}{
```
Comment (butonic) [inline pkg/command/storageeos.go:111]: getting rid of unnecesarry config options is ok: https://github.com/owncloud/ocis-reva/issues/155

File: pkg/command/storageeosdata.go [hunk 2]
```diff
@@ -110,24 +110,6 @@ func StorageEOSData(cfg *config.Config) *cli.Command {
 										"enable_home":            cfg.Reva.Storages.EOS.EnableHome,
 										"user_layout":            cfg.Reva.Storages.EOS.Layout,
 									},
-									"local": map[string]interface{}{
```
Comment (butonic) [inline pkg/command/storageeosdata.go:113]: getting rid of unnecesarry config options is ok: #155

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ pkg/command/storageeos.go  -2
+ pkg/command/storageeosdata.go  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=5426902&bid=17796625)

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ pkg/command/storageeos.go  -2
+ pkg/command/storageeosdata.go  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=5426902&bid=17796625)

**Pull request merged**